src/Common/src/System/Drawing/KnownColorTable.cs

```diff
@@ -59,9 +56,17 @@ private static void InitColorTable()
         {
             int[] values = new int[(unchecked((int)KnownColor.MenuHighlight)) + 1];
 
-#if FEATURE_SYSTEM_EVENTS
-            SystemEvents.UserPreferenceChanging += new UserPreferenceChangingEventHandler(OnUserPreferenceChanging);
-#endif
+            Type systeEventsType = Type.GetType("Microsoft.Win32.SystemEvents, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089", throwOnError: false);
+            EventInfo upEventInfo = systeEventsType?.GetEvent("UserPreferenceChanging", BindingFlags.Public | BindingFlags.Static);
+
+            if (upEventInfo != null)
+            {
+                Type userPrefChangingDelegateType = Type.GetType("Microsoft.Win32.UserPreferenceChangingEventHandler, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089", throwOnError: false);
+                MethodInfo mi = Type.GetType("System.Drawing.KnownColorTable")?.GetMethod("OnUserPreferenceChanging", BindingFlags.NonPublic | BindingFlags.Static);
```

Comment:
Instead of doing `Type.GetType("System.Drawing.KnownColorTable")` can't we just do: `GetType()` since this code is in `KnownColorTable` I don't think we need to use `Type.GetType` api.